css/style.css

```diff
@@ -1039,22 +1041,1372 @@ li { list-style-type: none; }
+    line-height: 16px;
+    font-weight: bold;
+    text-align: center;
+    text-decoration: none;
+    color: #000;
+    cursor: pointer;
+    padding: 0;
+    border: 0;
+  }  
+
+  .test-drive__button {
+    display: flex;
+    width: 160px;
+    height: 40px;
+    align-items: center;
+    justify-content: center;
+    font-size: 13px;
+    text-align: center;
+    font-weight: bold;
+    border: 0;
+    color: #fff;
+    background-color: #f00;
+  }
+  
+  .test-drive__button:hover {
+    font-size: 14px;
+    color: #fff;
+    border: 3px solid #fff;
+    background-color: #f00;
+    cursor: pointer;
+    font-weight: bold;
+    padding: 0;
+  }
+  
+  .test-drive__button:focus {
+    display: flex;
+    color: #fff;
+    border: 1px solid #fff;
+    background-color: #f00;
+    cursor: pointer;
+    font-weight: bold;
+    padding: 0;
+    outline: none;
+    box-shadow: 0 0 0 3px;
+  }
+  
+  .test-drive__button:active {
+    background-color: #fff;
+    border-color: #000;
+    color: #000;
+    font-size: 12px;
+  }
+  
+  .section__btn-pointer {
+    display: flex;
+    width: 360px;
+    height: 14px;
+    padding: 0;
+    margin: 0 auto ;
+    box-shadow: none;
+    background: url(https://i.ibb.co/GFX8Xs7/Vector-1.png) no-repeat;
+    background-size: 7%;
+    background-position: center center;
+    outline: none;
+  }
+
+  /* -------2-------- */
+
+  .sections__dashboard {
+    display: flex;
+    flex-flow: column wrap;
+    max-width: 375px;
+    height: 100%;
+    align-content: flex-end;
+    margin: 0;
+    background: url(https://i.ibb.co/JkTYWY9/2013-tesla-model-s-cockpit-i-pad.png) no-repeat;
+    background-size: 137%;
+    background-position: top left -58px;
+  }
+  
+  .dashboard__header {
+    display: none;
+  }
+  
+  .dashboard-right {
+    display: flex;
+    flex-flow: row wrap;
+    max-width: 375px;
+    max-height: 487.21px;
+    align-items: center;
+    justify-content: center;
+    font-size: 40px;
+    color: white;
+  }
+  
+  .dashboard-right__sidebar {
+    display: flex;
+    flex-flow: row wrap;
+    align-content: flex-start;
+    justify-content: flex-start;
+  }
+
+  .sidebar {
+    display: flex;
+    flex-flow: row wrap;
+    align-content: flex-start;
+    justify-content: flex-start;
+    width: 375px;
+    max-height: 300px;
+    color: #fff;
+    background: #f00;
+    background-size: 100%;
+    margin: 325px 0 0;
+  }
+  
+  .sidebar__symbol {
+    display: none;
+  }
+
+  .sidebar__title {
+    display: flex;
+    justify-content: center;
+    width: 316px;
+    height: 37px;
+    color: #fff;
+    margin: 18px 0 9px 27px;
+  }
+
+  .sidebar__item {
+    display: flex;
+    align-items: flex-start;
+    justify-content: center;
+    width: 316px;
+    height: 206px;
+    padding: 0;
+    margin: 0 0 0 27px;
+  }
+
+  /* ---ссылка тест и кнопка заказать--- */
+
+  .info {
+    display: none;
+  }
+
+  .sidebar__info {
+    display: none;
+  }
+
+   /* -------3-------- */
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Hi-Tesla/pullRequest?prid=5851407)